modules/ROOT/pages/cloudhub-connector.adoc

```diff
@@ -12,27 +14,21 @@ Create custom notifications inside your flows and exception strategies, as well
 
 Review the technical reference linked at top to learn how to manage applications and tenancies through the CloudHub connector.
 
-MuleSoft maintains this connector under the xref:anypoint-connectors.adoc#connector-categories[_Select_] support policy.
-
-
-== Installing from Anypoint Exchange
-
-To deploy, start, stop, and update applications as well as send notifications from your application to CloudHub, get the https://www.anypoint.mulesoft.com/exchange/?type=connector&search=cloudhub[CloudHub connector] from Anypoint Exchange.
-
-== Installing from Studio
-
-To access the CloudHub connector from Anypoint Studio, click *Help* > *Install New Software*. In *Work with*, click `Anypoint Connectors Update Site`. In the *Community* category, click *CloudHub Connector*. If this option is not listed, click *Help* > *Installation Details* and see if the connector was previously installed.
 
-Click *Next* > then *Next* again > *I accept the terms of the license agreement*  > *Finish*, and restart Studio.
+== Install from Anypoint Exchange
 
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. Click *Login* in Anypoint Exchange and supply your Anypoint Platform username and password.
```

Comment:
"Anypoint Exchange" or "Exchange"? Latter used first, followed in second step by former.